templates/firefox-ios.xliff

```diff
@@ -2116,16 +2199,16 @@
         <source>Update</source>
         <note>Button to update the user's password</note>
       </trans-unit>
-      <trans-unit id="Upgrade Firefox to connect.">
-        <source>Upgrade Firefox to connect.</source>
+      <trans-unit id="Upgrade Firefox to connect">
+        <source>Upgrade Firefox to connect</source>
```

Comment:
Is there a way that this can be marked as fuzzy in Pontoon, instead of having to re-translate the whole thing? Or maybe translation memory keeps it?
@flodolo probably can tell me :)

Reply:
Between 12 and 13 I want to send out a proposal on how to move all the strings to having a proper ID. Probably with some renaming magic to make sure we don't lose any for which we have to do work again.

@Delphine  Can we leave these like this for now?

Reply:
Since we're only talking about 2 strings, let's just do that for now.